### PR TITLE
add quick translation tool

### PR DESCRIPTION
**Pull request by jamesduncan**

### CODE REVIEW

**Comment by jamesduncan** [comment]
There is an issue with the link to the views to see the context of the UI elements...I will fix that tomorrow and then ask for a review.

**Comment by jamesduncan** [comment]
There are still some views (sub pages that are not popups) that you cannot quickly jump to with the link on the right. However I am happy to log it as a known issue and follow back another time because I don't know if I know the right way to fix it.
Screen Shot 2022-10-13 at 11 05 50 AM

**Comment by jamesduncan** [comment]
Okay...I took the time to rethink how to go about loading the views for the in context view of the translation. I was able to put code back to its original place in a couple of the files so there isn't any code that is hacking a solution anymore. This is ready for review and has passed all tests (in that it doesn't break any previous tests but we could consider writing translation tool tests now that this has been migrated into v2).

File: ui/portal_translation_tool.js [hunk 1]
```diff
@@ -0,0 +1,976 @@
...
+
+   ui() {
+      const translationToolBody = {
+         view: "scrollview",
+         css: "lightgray ab_amp",
+         body: {
+            rows: [
+               {
+                  view: "accordion",
+                  id: "qtt_accordion_" + this.containerDomID,
```
Comment (nh758) [inline ui/portal_translation_tool.js:17]: I think in general we're trying to use string literal syntax, but not a big deal to me.
```js
id: `qtt_accordion_${this.containerDomID}`,
```

**Review by nh758 — APPROVED** [review]

File: ui/portal_translation_tool.js (hunk 1 above)
Comment (jamesduncan) [reply ui/portal_translation_tool.js]: Okay I switched all items I could find to use literal syntax.

**Pull request merged**